seatunnel-engine/seatunnel-engine-storage/imap-storage-api/src/test/java/org/apache/seatunnel/engine/imap/storage/api/ProtoStuffSerializerTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.apache.seatunnel.engine.imap.storage.api;
+
+import org.apache.seatunnel.engine.imap.storage.api.common.ProtoStuffSerializer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ProtoStuffSerializerTest {
+
+    @Test
+    public void testProtoStuffSerializerForArrayType() {
+        System.setProperty("protostuff.runtime.preserve_null_elements", "true");
```

Comment:
Before requesting a review, you should check the corresponding reviews first, otherwise it doesn't make any sense.

Reply:
You should automatically do these parameter injections if needed.

Reply:
> Before requesting a review, you should check the corresponding reviews first, otherwise, it doesn't make any sense.

Sorry, my fault, I will fix these.

Reply:
> You should automatically do these parameter injections if needed.

I tried injecting this global property in a static method, but it doesn't seem to work.

Reply:
> > You should automatically do these parameter injections if needed.
> 
> I tried injecting this global property in a static method, but it doesn't seem to work.

![image](https://user-images.githubusercontent.com/48236177/207772169-c7c0959a-656d-4db6-8c93-58282482bd82.png)
![image](https://user-images.githubusercontent.com/48236177/207772510-e911e6b8-8f13-4d48-be6d-4962956ac66e.png)

Reply:
> You should automatically do these parameter injections if needed.

Done.